bindings/python/src/datetime.cpp

```diff
@@ -55,6 +56,33 @@ struct time_duration_to_python
     }
 };
 
+struct time_point_to_python
+{
+    static PyObject* convert(lt::time_point tpt)
+    {
+        object result;
+        if (tpt > lt::min_time()) {
+            lt::time_point now = lt::clock_type::now();
+            time_t const tm = lt::total_seconds(tpt - now);
```

Comment:
hm.. I don't see how this could be correct... you're just looking at the number of seconds from now.. std::localtime() expects the number of seconds since 00:00 january 1st, 1970